pydis_site/apps/content/resources/guides/python-guides/subclassing_context.md

```diff
@@ -0,0 +1,179 @@
+
+    @discord.ui.button(label="Confirm", style=discord.ButtonStyle.green)
+    async def confirm(
+        self, button: discord.ui.Button, interaction: discord.Interaction
+    ):
+        self.value = True  # returns True if the confirm button is pressed
+        await interaction.response.defer()
+        if self.delete_after:
+            await interaction.delete_original_message()
+        self.stop()  # stops the view and deletes the confirmation message
+
+    @discord.ui.button(label="Cancel", style=discord.ButtonStyle.red)
+    async def cancel(self, button: discord.ui.Button, interaction: discord.Interaction):
+        self.value = False  # returns False if the cancel button is pressed
+        await interaction.response.defer()
+        if self.delete_after:
+            await interaction.delete_original_message()
+        self.stop()  # stops the view and deletes the confirmation
+
+
+# We have made two buttons, one for Confirm and one for Cancel.
+
+
+class CustomContext(commands.Context):
+    # create new Context methods here. For example:
+
+    async def send_embed(self, message, **kwargs):
+        return await self.send(
+            embed=discord.Embed(
+                description=message, colour=discord.Color.blurple(), **kwargs
+            )
+        )
+        # This is a shortcut method to sending a custom embed. Pre-set values can be put here, for example the color kwarg in the example above means it has a pre-set color of blurple
+
+    async def send_error_embed(self, message, **kwargs):
+        return await self.send(
+            embed=discord.Embed(
+                description=message, colour=discord.Color.red(), **kwargs
+            )
+        )
+        # embed with pre-set color of red
+
+    async def send_sucess_embed(self, message, **kwargs):
+        return await self.send(
+            embed=discord.Embed(
+                description=message, colour=discord.Color.green(), **kwargs
+            )
+        )
+        # embed with pre-set color of green
+
+    # our custom prompt function using buttons
+    async def prompt(
+        self,
+        message: str,
+        *,
+        timeout: float = 60.0,
+        delete_after: bool = True,
+        author_id: Optional[int] = None,
+    ) -> Optional[bool]:
+        """
+        Parameters
+        message: str
+            The message to show along with the prompt.
+
+        timeout: float (Defaults to 60 seconds)
+            How long to wait before stopping the view
+        delete_after: bool (Defaults to True)
+            Whether to delete the confirmation message after we're done.
+
+        author_id: Optional[int] (Defaults to ctx.author)
+            The member who should respond to the prompt.
+        -------
+        Returns
+            True if the confirm button is pressed,
+            False if cancel button is pressed,
+            None if automatically denied due to no response
+        """
```

Comment:
This docstring is missing a description other than the reference of parameters.

Additionally, the typings are present in the definition so you don't need to repeat them in the docstring.